tomte/cli.py

```diff
@@ -93,6 +96,12 @@ def freeze_dependencies(output_path: Optional[str]) -> None:
     """Freeze dependencies."""
     freeze_dependencies_main(output_path=output_path)
 
+@click.command()
+def check_spelling() -> None:
+    """Check spelling on all the doc .md files."""
+    script_path = Path(__file__).resolve()
+    target_script = Path(script_path.parent, 'scripts', 'check_spelling.sh')
+    subprocess.call(['sh', target_script])
```

Comment:
Not quite what I want. The `check_spelling.sh` should be part of this tool.

Only the configuration `.spelling` should reside in the users repo

Reply:
Ah sorry, got it now